lib/config.js

```diff
@@ -12,6 +12,7 @@ var version = require('../package').version
 var packageName = require('../package').name
 
 const { WildcardMatcher } = require('./wildcard-matcher')
+const { getCloudMetadata } = require('./instrumentation/cloud-metadata')
```

Comment:
Debated this one myself internally -- my thinking way it sort of does since we're "instrumenting" the environment to fetch this data even if it's not a traditional "monkey patching stuff to get timing instrumentation".  

But since I wasn't 100% sold on it myself and it also raised your eyebrow I'll pull it back up into the directory above.